media_utils/video_puppeteer.py

```diff
@@ -187,12 +198,18 @@ def playback_done(video):
     # either still playing or there is a problem, so this will never
     # return True.
 
+    active_re = re.compile('NextGetSampleIndex:([\d]+)')
+
     debug_lines = video.get_debug_lines()
     if debug_lines is not None:
+        for occurence in active_re.findall(' '.join(debug_lines)):
+            if occurence.group(1) == '0':
+                raise VideoException('Video %s is stalled' % video)
```

Comment:
My take:
*Just seeing that `NextGetSampleIndex==0` should not result in a test failure. We should also check whether the video is actually stalled. 
*`NextGetSampleIndex==0` does not imply that the video is stalled.